pdc/apps/compose/views.py

```diff
@@ -468,35 +467,19 @@ def filter_queryset(self, qs):
             return sorted(qs)
         return qs
 
-    @staticmethod
-    def _filter_rpm_value(query_params, query_param, rpm_key, qs, q):
-        rpm_value = query_params.get(query_param, None)
-        s = 'variant__variantarch__composerpm__rpm__' + rpm_key
-        if rpm_value is None:
-            return qs, q
-        if len(rpm_value) > 1:
-            condition_list = []
-            for val in rpm_value:
-                condition_list.append(Q(**{s + '__iexact': val}))
-            qs = qs.filter(reduce(operator.or_, condition_list)).distinct()
-            return qs, None
-        if len(rpm_value) == 1:
-            q &= Q(**{s + '__iexact': rpm_value[0]})
-
-        return qs, q
-
     def _filter_nvras(self, qs):
         q = Q()
-        query_params = dict(self.request.query_params.iterlists())
-        query_parma_rpm_key_mapping = [('rpm_name', 'name'),
+        query_params = self.request.query_params
+        query_param_rpm_key_mapping = [('rpm_name', 'name'),
                                        ('srpm_name', 'srpm_name'),
                                        ('rpm_version', 'version'),
                                        ('rpm_release', 'release'),
                                        ('rpm_arch', 'arch')]
-        for query_param, rpm_key in query_parma_rpm_key_mapping:
-            qs, q = self._filter_rpm_value(query_params, query_param, rpm_key, qs, q)
-            if q is None:
-                return qs
+        for query_param, rpm_key in query_param_rpm_key_mapping:
+            rpm_value = query_params.get(query_param, None)
+            s = 'variant__variantarch__composerpm__rpm__' + rpm_key
+            if rpm_value:
+                q &= Q(**{s + '__iexact': rpm_value})
```

Comment:
If user input multiple values for one field, it will return empty queryset, right? If so, should it be documented?

Another thing is if you already handled these fields here, should them be popped from `self.request.query_params` to avoid be handled in filter part again?